arctic/store/version_store.py

```diff
@@ -44,6 +44,10 @@ def initialize_library(cls, arctic_lib, hashed=True, **kwargs):
         if '%s.changes' % c.name not in mongo_retry(c.database.collection_names)():
             # 32MB buffer for change notifications
             mongo_retry(c.database.create_collection)('%s.changes' % c.name, capped=True, size=32 * 1024 * 1024)
+        # TODO: This should not be here
+        if '%s.metadata' % c.name not in mongo_retry(c.database.collection_names)():
```

Comment:
.metadata collection doesn't exist at the moment. Do I need to move this code to somewhere else (eg when the mongo library is created)?